server/app.js

```diff
@@ -2,6 +2,7 @@ const express = require('express');
 const { join } = require('path');
 const compression = require('compression');
 const cookieParser = require('cookie-parser');
+const routes = require('./routes');
```

Comment:
just add an empty line between third party module and your module to make the code clear.